arangod/Scheduler/Scheduler.h

```diff
@@ -242,6 +242,22 @@ class Scheduler {
     });
   }
 
+  // Yield the current thread
+  futures::Future<futures::Unit> yield(
```

Comment:
No objections against this method, but is it being used in this PR?

Reply:
It was initially used in `statusChangeWithTimeout` instead of the `std::this_thread::yield()`. But then I figured that the loop in that function is essentially a busy loop and that it might not be the best idea to hammer on the scheduler. That is why I introduce the delay. 

In particular this delay only happens if someone aborts or commits a transaction, while there is also an ongoing request _using_ that transaction.